editors/sc-ide/core/sc_server.hpp

```diff
@@ -163,9 +170,9 @@ private slots:
     QAction * mActions[ActionCount];
 
     VolumeWidget *mVolumeWidget;
-    QTimer mRecordTimer;
-    boost::chrono::system_clock::time_point mRecordTime;
+    int mRecordTime;
```

Comment:
what is `mRecordTime`? it used to be a timepoint, now it is an int ... of what? apples? seconds? days?